src/query/ast/tests/it/testdata/statement.txt

```diff
@@ -304,6 +304,46 @@ DescribeTable(
 Some(
     "TabSeparatedWithNamesAndTypes",
 )
+---------- Input ----------
+describe "name""with""quote";
+---------- Output ---------
+DESCRIBE "name"with"quote"
```

Comment:
the display may cause bugs when persisting the SQL into a view. We've to escape it at the display stage.